### ISSUE TITLE
Model can't be found in RESTactions trait

### ISSUE THREAD

**Issue by DarioSoller** [webNeat/lumen-generators#67]
Hi,
I am having an issue, that the generated RESTaction.php trait does not get the model everytime it tries to get it via `$m = self::MODEL;`.

The corresponding generated controller looks like this:
```php
<?php namespace App\Http\Controllers;

use App\Spots;

class SpotsController extends Controller {

    const MODEL = "App\Spots";

    use RESTActions;

}
```

Stacktrace, when calling the endpoint:
![image](https://user-images.githubusercontent.com/2143409/54232775-c2670900-450b-11e9-85f4-2a5d591b2727.png)

Files (with the Spot Model selected):
![image](https://user-images.githubusercontent.com/2143409/54233291-f4c53600-450c-11e9-8fd3-6e3d74802fa1.png)

Versions:
```
"name": "laravel/lumen-framework",
"version": "v5.7.7",
```
```
"name": "wn/lumen-generators",
"version": "1.3.4",
```

I am kind of a beginner with lumen so maybe I am missing anything in using the generators or setting it all up. Therefore it's still hard for me to properly debug the php code. I am thankful for any tips regarding this issue and will of course provide more infos if I can.

**Comment by eschaar** [comment]
If I'm not mistaken, your file for the model is called Spot (Spot.php) and not Spots.

**Comment by DarioSoller** [comment]
@eschaar: Ahhh you're absolutely right! The problem is that with the plural of the Model. So one can maybe say plural is the new "off by one" error :)

But as I said, I am new to it, so I used the command `php artisan wn:controller spots` because I also wanted to have an API endpoint which is `/spots` and a `SpotsController`. I did not know that `php artisan wn:controller spot` would generate me an `SpotsController`, as well, but adds singular model reference and a singular API endpoint. So this seems to be a little hidden inconsistency, right!? 

I already had the model so did not use something like `php artisan wn:resource spot`, which would've done it right. So maybe one could add to the docs, to better use singular nouns for generating stuff.

I will close this issue. Thanks for pointing me into the obvious right direction!
